lib/mix/lib/mix/tasks/test.ex

```diff
@@ -14,17 +30,91 @@ defmodule Mix.Tasks.Test do
           Mix.raise("Failed to cover compile directory: " <> compile_path)
       end
 
-      output = opts[:output]
+      formatters = Keyword.get(opts, :formatters, @default_formatters)
 
       fn ->
+        {:result, ok, _fail} = :cover.analyse(:coverage, :module)
         Mix.shell().info("\nGenerating cover results ...")
```

Comment:
Let's print this before we call analyze, otherwise the user may wait for something and they don't know what it is.